kore/src/Kore/Step/Simplification/Condition.hs

```diff
@@ -44,7 +44,7 @@ import qualified Kore.Internal.Substitution as Substitution
 import Kore.Rewriting.RewritingVariable (
     RewritingVariableName,
  )
-import qualified Kore.Step.Simplification.Predicate as Predicate
+import {-# SOURCE #-} qualified Kore.Step.Simplification.Predicate as Predicate
```

Comment:
This resolves a cyclic dependency in the import graph. The cycle will be resolved when the `TermLike` simplifier is finally fixed so that it does not call the `Condition` simplifier.